File: 2_ruby/es_palindromo2.rb
def es_palindromo2?(palabra)
  izquierdo = 0
  derecho = palabra.length - 1
  while izquierdo < derecho
    if palabra[izquierdo].downcase != palabra[derecho].downcase
      return false
    end
    izquierdo += 1
    derecho -= 1
  end
  return true
end

puts es_palindromo2?("mex")
puts es_palindromo2?("Lol")

File: 8_php/es_palindromo2.php
<?php
function es_palindromo2($palabra){
  $izquierdo = 0;
  $derecho = strlen($palabra) - 1;
  while($izquierdo < $derecho){
    if (strtolower($palabra[$izquierdo]) != strtolower($palabra[$derecho])){
      return False;
      $izauierdo++;
      $derecho--;
    }
    return True;
  }
}

echo es_palindromo2("mex");
echo es_palindromo2("Lol");
?>
